src/CBot/CBotToken.h

```diff
@@ -131,19 +131,19 @@ class CBotToken : public CBotDoublyLinkedList<CBotToken>
     /**
      * \brief Return the beginning location of this token in the original program string
      */
-    int GetStart();
+    std::size_t GetStart() const;
 
     /**
      * \brief Return the ending location of this token in the original program string
      */
-    int GetEnd();
+    std::size_t GetEnd() const;
 
     /**
      * \brief SetPos Set the token position in the CBot program
      * \param start The start position of the token
      * \param end The end position of the token
      */
-    void SetPos(int start, int end);
+    void SetPos(const std::size_t start, const std::size_t end);
```

Comment:
Like someone said already, putting `const` in header files does more harm than good. The caller doesn't care since the argument is passed by value anyway, yet it restricts the implementation for no good reason.